NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Pp create post Ticket #6

# Description
As an author, I would like to be able to create Posts so I can share my thoughts with the world. Once a user has logged in, they can now select a "new posts" button in the post screen. This opens a new page to add details about a new post. After required info is entered, a user can save their post and will be taken to the newly created details page for that post.

## Type of change
Please delete options that are not relevant.



# Testing Instructions
Pull down and checkout branch pp-create-post
Start both server side in VS and and client side from the client folder
use foo@bar.com and password: ****** to log in to the application and see the home page
Once logged in navigate to the the menubar 
click the new "new post" button at the top
That should open a new page with form fields to fill out for a new post
Enter some information, so of the fields are required
Once enough information is entered, click the button to save
If successfully saved, the page will redirect to a link for the details of that new post (details page is TBA, see Ticket #5)

# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [ x I have added test instructions that prove my fix is effective or that my feature works
